core/src/main/resources/error/error-classes.json

```diff
@@ -4276,4 +4293,4 @@
       "Not enough memory to build and broadcast the table to all worker nodes. As a workaround, you can either disable broadcast by setting <autoBroadcastjoinThreshold> to -1 or increase the spark driver memory by setting <driverMemory> to a higher value<analyzeTblMsg>"
     ]
   }
-}
+}
```

Comment:
This can be reverted?

Reply:
I can check if the tests pass if this line is removed. I had some challenges with failing tests because they check the style of the his json file :(